content/radars/2021-09-devsecops.yml

```diff
@@ -0,0 +1,203 @@
+    twitter: k_gamanji
+    linkedin: katie-gamanji
+    title: Ecosystem Advocate at CNCF
+  - name: Sergiu Petean
+    photo: https://www.cncf.io/wp-content/uploads/2021/09/Sergio-Patean.jpeg
+    twitter: PeteanuSergiu
+    linkedin: peteanusergiu
+    title: Head of DevOps at Allianz Direct
+  - name: Keith Nielsen
+    photo: https://www.cncf.io/wp-content/uploads/2021/09/Keith-Nielsen.jpeg
+    title: Director Cloud Architecture at Discover Financial Services
+points:
+  - name: Artifactory 
+    homepage: https://jfrog.com/artifactory/
+    level: adopt
+    votes: 16
+      adopt: 14
+      trial: 1
+      hold: 1
+  - name: Sonarqube
+  - homepage: https://www.sonarqube.org/
+    level: adopt
+    votes: 15
+      adopt: 13
+      trial: 2
+      assess: 0
+  - name: XRay
+  - homepage: https://www.getxray.app/
+    level: trial
+    votes: 10
+      adopt: 1
+      trial: 3
+      assess: 6
+   - name: GitHub Actions
+   - homepage: https://github.com/actions
+    level: assess
+    votes: 12
+      adopt: 4
+      trial: 3
+      assess: 5
+   - name: Calico/Tigera
+   - homepage: https://www.tigera.io/tigera-products/calico/
+    level: adopt
+    votes: 10
+      adopt: 9
+      trial: 0
+      assess: 1
+   - name: OPA
+   - homepage: https://www.openpolicyagent.org/
+    level: adopt
+    votes: 12
+      adopt: 6
+      trial: 4
+      assess: 2
+   - name: Hashicorp Sentinel
+   - homepage: https://www.hashicorp.com/sentinel
+    level: assess
+    votes: 4
+      adopt: 1
+      trial: 1
+      assess: 2
+    - name: Terraform
+    - homepage: https://www.terraform.io/
+    level: adopt
+    votes: 20
+      adopt: 18
+      trial: 0
+      assess: 1
+    - name: Hashicorp Vault
+    - homepage: https://www.vaultproject.io/
+    level: adopt
+    votes: 18
+      adopt: 14
+      trial: 3
+      assess: 0
+    - name: Istio
+    - homepage: https://istio.io/
+    level: adopt
+    votes: 16
+      adopt: 6
+      trial: 1
+      assess: 7
+    - name: Linkerd
+    - homepage: https://linkerd.io/
+    level: assess
+    votes: 5
+      adopt: 2
+      trial: 0
+      assess: 3
+    - name: ArgoCD
+    - homepage: https://argoproj.github.io/argo-cd/
+    level: adopt
+    votes: 11
+      adopt: 8
+      trial: 1
+      assess: 1
+    - name: Harness
+    - homepage: https://harness.io/
+    level: assess
+    votes: 6
+      adopt: 2
+      trial: 2
+      assess: 2
+    - name: Cilium
+    - homepage: https://cilium.io/
+    level: assess
+    votes: 7
+      adopt: 2
+      trial: 2
+      assess: 3
+   - name: Trivy
+   - homepage: https://aquasecurity.github.io/trivy/v0.17.0/
+    level: assess
+    votes: 7
+      adopt: 0
+      trial: 4
+      assess: 3
+    - name: Sonatype Nexus
+    - homepage: https://www.sonatype.com/products/repository-pro
+      level: assess
+    votes: 10
+      adopt: 6
+      trial: 1
+      assess: 0
+companies:
```

Comment:
The list only includes 23 organizations. We had 24 end users orgs that submitted their votes. Seems like one is missing from this list

Reply:
resolved on slack